core/rid.h

```diff
@@ -61,6 +59,8 @@ class RID {
 public:
 	_FORCE_INLINE_ RID_Data *get_data() const { return _data; }
 
+	//_FORCE_INLINE_ void erase(RID &p_rid) { _data = nullptr; } // Useful to have?
```

Comment:
Would it be helpful to have this function?
I don't really need it as the friend, RID_OwnerBase has access to clear the RID.